docs/kbs_attestation_protocol.md

```diff
@@ -108,113 +146,303 @@ attestation challenge to KBC. The payload format is as follows:
 
 - `nonce`
 
-The fresh number passed to KBC. KBC needs to place it in the evidence sent to KBS in the next 
-step to prevent replay attacks.
+The fresh number passed to KBC. KBC needs to place it in the evidence sent to
+the KBS in the next step to prevent replay attacks.
 
 - `extra-params`
 
-The reserved extra parameter field which is used to pass the additional information provided by 
-the KBS when some specific HW-TEE needs to be attested.
+The reserved extra parameter field which is used to pass the additional
+information provided by the KBS when some specific HW-TEE needs to be attested.
 
-## Attestation
+## `Attestation`
 
-After receiving the invitation challenge, KBC collects the attestation evidence from the HW-TEE 
-platform and organizes it into the following payload format:
+After receiving the attestation challenge, the KBC builds an attestation
+evidence from the HW-TEE platform and organizes it into the following payload:
 
 ```json
 {
     "evidence": {
-        /* The nonce in the Challenge; its hash needs to be included in HW-TEE evidence and signed by HW-TEE hardware. */
+        /*
+         * The nonce in the Challenge; its hash needs to be included in HW-TEE
+         * evidence and signed by HW-TEE hardware.
+         */
         "nonce": "",
```

Comment:
Is it worth changing lines like this to something like:

```suggestion
        "nonce": "$nonce",
```

... since currently if you don't read the comments and only look at the JSON, it suggests this field is / should be blank.

Reply:
Fixed, thanks.